packages/yarnpkg-pnp/webpack.config.hook.js

```diff
@@ -1,5 +1,7 @@
+require(`@yarnpkg/monorepo/scripts/setup-ts-execution`);
```

Comment:
`@yarnpkg/builder/sources/tools/makeConfig` is a ts file so need to setup `@babel/register`